traveltimepy/mapper.py

```diff
@@ -138,9 +140,10 @@ def create_time_map_fast(
                         coords=cur_coordinates,
                         transportation=transportation,
                         travel_time=travel_time,
-                        arrival_time_period="weekday_morning",  # TODO: make customizable with enum / literal
+                        arrival_time_period="weekday_morning",
```

Comment:
Added these comments in a previous PR, but I didn't notice, that "weekday_morning" is the only possible period, according to docs. Removing the TODOs